jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildDockerMojo.java

```diff
@@ -90,6 +91,7 @@ public void execute() throws MojoExecutionException {
               null,
               null,
               mavenHelpfulSuggestionsBuilder.build());
+      ProxyProvider.init(getSession().getSettings());
```

Comment:
These proxy settings may contain encrypted values which will need to be decrypted.  It would likely be better to add support to `MavenSettingsServerCredentials` and handle these proxy values in PluginConfigurationProcessor.processCommonConfiguration*`

Reply:
I agree we need decryption, but I guess this is a good start working in a very basic case. We can implement decryption in a later PR.

Reply:
Filed https://github.com/GoogleContainerTools/jib/issues/1369.